examples/basic/verifying_md.py

```diff
@@ -0,0 +1,209 @@
+import desbordante
+import pandas as pd
+
+from typing import TypedDict
+from desbordante.md import ColumnSimilarityClassifier
+from desbordante.md.column_matches import Levenshtein, Custom
+
+GREEN_CODE = "\033[1;42m"
+RED_CODE = "\033[1;41m"
+BLUE_CODE = "\033[1;46m"
+DEFAULT_COLOR_CODE = "\033[1;49m"
+
+
+class MDParams(TypedDict):
+    lhs: list[ColumnSimilarityClassifier]
+    rhs: ColumnSimilarityClassifier
+
+
+def print_results(verifier):
+    if verifier.md_holds():
+        print(GREEN_CODE, "MD holds", DEFAULT_COLOR_CODE, "\n")
```

Comment:
This outputs
```
 MD holds 

```
Perhaps you want
```suggestion
        print(f"{GREEN_CODE}MD holds{DEFAULT_COLOR_CODE}\n")
```
for ```
MD holds
```